dev_docs/operations/packages_idm.mdx

```diff
@@ -41,62 +41,62 @@ Every package has:
   - a type, see [Package Types](#package-types)
   - dependencies on other packages in the repo or from NPM declared in their `kibana.jsonc` file and updated automatically
   - configuration files like `BUILD.bazel`, `jest.config.js`, `package.json`, etc. which are automatically generated when necessary from the `kibana.jsonc` file.
-  - a single interface. When you import a package you get it's one and only interface. Packages don't have sub-imports, or sub-modules.
+  - a single interface. When you import a package you get its one (and only) interface. Packages don't have sub-imports or sub-modules.
```

Comment:
This is the only required change. Using "it's": 

> When you import a package you get _it is_ one and only interface.

Reply:
I'm so bad at its and it's, thanks for your attention to detail.